testing/jormungandr-automation/src/jormungandr/explorer/mod.rs

```diff
@@ -52,22 +53,49 @@ pub struct ExplorerProcess {
 }
 
 impl ExplorerProcess {
-    pub fn new(node_address: String, logs_dir: Option<std::path::PathBuf>) -> Self {
+    pub fn new(
+        node_address: String,
+        logs_dir: Option<std::path::PathBuf>,
+        params: ExplorerParams,
+    ) -> Self {
         let path = get_explorer_app();
         let explorer_port = get_available_port();
         let explorer_listen_address = format!("127.0.0.1:{}", explorer_port);
 
+        let mut explorer_cmd = Command::new(path);
+        explorer_cmd.args(&[
+            "--node",
```

Comment:
do you think it's convenient to have all this string into a struct / whatever so that they can be changed in one place rather than having to dig through the code?

Reply:
Yes you are right, all this parameters will have to go into the ExplorerParams struct, I will do it in the next PR